deprecating the PaloAltoNetworks content pack published in market place. (#29604)
demisto/content: Packs/PaloAltoNetworks_IoT3rdParty/README.md

 - Coverting an asset to external format.
 - Reporting a status back to Palo Alto Networks IoT cloud.

> This content pack is deprecated and here is the document on how to get the latest Palo Alto Networks IoT 3rd Party Integrations Content Pack https://docs.paloaltonetworks.com/iot/iot-security-integration/get-started-with-iot-security-integrations/third-party-integrations-using-a-full-featured-xsoar-server . 

Reviewer: ```suggestion
This content pack is deprecated. To get the latest Palo Alto Networks IoT 3rd Party Integrations content pack, visit https://docs.paloaltonetworks.com/iot/iot-security-integration/get-started-with-iot-security-integrations/third-party-integrations-using-a-full-featured-xsoar-server. 
```
